_maps/RandomRuins/SpaceRuins/LandmineVault.dmm

```diff
@@ -0,0 +1,391 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/turf/template_noop,
+/area/template_noop)
+"b" = (
+/turf/open/floor/plating/airless,
+/area/ruin/space)
+"c" = (
+/turf/closed/wall/r_wall,
+/area/ruin/space)
+"d" = (
+/obj/effect/decal/fakelattice,
+/turf/template_noop,
+/area/template_noop)
+"e" = (
+/obj/structure/lattice,
+/turf/template_noop,
+/area/template_noop)
+"f" = (
+/obj/effect/decal/cleanable/ash,
+/turf/open/floor/plating/airless,
+/area/ruin/space)
+"j" = (
+/obj/effect/decal/cleanable/dirt/dust,
+/turf/open/floor/plating/airless,
+/area/ruin/space)
+"m" = (
+/obj/effect/decal/cleanable/dirt,
+/turf/open/floor/plating/airless,
+/area/ruin/space)
+"p" = (
+/obj/structure/safe,
+/obj/item/book/granter/crafting_recipe/dusting/laser_musket_prime,
```

Comment:
I believe it has been said on numerous occasions that mapping in or making it easier/more reliable to obtain something rare or difficult to acquire is not a good thing, and this isn't an exception. Please don't map in the prime crafting books.

Reply:
Sure